scripts/settings.toml

```diff
@@ -0,0 +1,34 @@
+[payment]
+pay_threshold = "10000000000000"
+close_threshold = "-10000000000000"
+close_fraction = "100"
+buffer_period = 3
+eth_address = "0x0101010101010101010101010101010101010101"
```

Comment:
Whoops, yes this is a duplicate of settings/test.toml... We can't use the rita_test.toml from the old test because some of the fields in that one (namely the wg priv key) prevent this version of spinning up rita from running properly